### PR TITLE
Grafana secret deployed from google secret manager

### PR DESCRIPTION
**Pull request by chaodaiG**
The secret currently need to be deployed manually if there is an update, make it backed up by google secret manager so that it can be automatically generated

### CODE REVIEW

File: config/prow/cluster/kubernetes_external_secrets.yaml [hunk 1]
```diff
@@ -27,3 +26,16 @@ spec:
...
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: grafana
+  namespace: prow-monitoring
+spec:
+  backendType: gcpSecretsManager
+  projectId: k8s-prow
+  data:
+  - key: gke_k8s-prow_us-central1-f_prow__prow-monitoring__grafana
```
Comment (chaodaiG) [inline config/prow/cluster/kubernetes_external_secrets.yaml:39]: This secret already exists in k8s-prow project, value was grabbed from the secret located in cluster

**Review by cjwagner — APPROVED** [review]

**Comment by cjwagner** [comment]
/hold
Will this work with the existing secret? I'd expect this to fail if a secret already exists and is not managed by KES.

**Comment by chaodaiG** [comment]
> /hold
> Will this work with the existing secret? I'd expect this to fail if a secret already exists and is not managed by KES.

I have tested this on my dev cluster, and KES just went ahead override existing secret. I think this is WAI, can prevent someone from manually messing up a secret

**Pull request merged**